### ISSUE TITLE
Reduce Tabs to 5 to align with platform UX guidelines on tabs

### ISSUE THREAD

**Issue by jpvajda** [newrelic/developer-website#1434]
* We need to have less Tabs, the platform UI is limited to 4 tabs we 7 total, let's try to get to 5 tabs. This needs design input before starting. 

### Ideas

  * Combine Visualizations and Nerd packs as a Viz is a nerdpack (_technically_)
  * Nerdpackss are out of scope for the MVP so we don't need to display this just yet
  * Put "Requirements" meta data  on the overview tab (_we need to confirm what will populate the requirements of a pack_)
  * Define what Requirements actually are, we assume it's based on the [instrumentation section of the config](https://github.com/newrelic/newrelic-observability-packs/blob/b80674e6594e5025b0803b644c585dd1df7f07bf/_template/config.yml#L22)

### Acceptance Criteria

- [X] Work with `Kike Alonzo` on a design and approach first 
- [x] Requirements tab is removed 
- [x] Instrumentation meta data appears in a new section below the `How to Use` right rail box.
- [x] Nerdpack tab is removed
- [x] Don't change synthetics to synthetic checks (that's from an old design)

### Mock Up
![#1434](https://user-images.githubusercontent.com/8354661/124609260-c30bd200-de6f-11eb-8d93-c15c504e6af9.jpg)

**Comment by kukukaka** [comment]
@jpvajda I added Requirements and Support (it was there on some of the designs) on the overview sidebar, and replaces Nerpacks with Viz. It follows the pattern we have on some other places on the site, so I think it could be a good option. 

I used the instrumentation example you shared for the requirements, but I think we would need some help or further explanations to make it understandable by the user, right?

![#1434](https://user-images.githubusercontent.com/8354661/124609260-c30bd200-de6f-11eb-8d93-c15c504e6af9.jpg)
